bigframes/dataframe.py

```diff
@@ -111,6 +113,9 @@ class DataFrame(vendored_pandas_frame.DataFrame):
     # Must be above 5000 for pandas to delegate to bigframes for binops
     __pandas_priority__ = 15000
 
+    # Type annotation for anywidget instance
+    _anywidget_instance: Optional[weakref.ReferenceType["TableWidget"]] = None
```

Comment:
I thought we discussed that having the DataFrame store a widget isn't necessary anymore?